### ISSUE TITLE
[4.0][a11y] Button incorectly marked as a link in Sample data module

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-cms#20997]
The Install button in Sample data module is incorrectly marked as a link. 
See:
* [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=111%2C412#info-and-relationships) (Level A)
* [G115](https://www.w3.org/TR/WCAG20-TECHS/G115.html): Using semantic elements to mark up structure
* [Links vs. Buttons in Modern Web Applications](https://marcysutton.com/links-vs-buttons-in-modern-web-applications/)

**Comment by brianteeman** [comment]
Sorry but this is of no use at all. Just sending a link to a generic page in the documentation is absolutely zero help at all.

What we need is specific information about what is not accessible and suggestions on how to make it accessible.

**Comment by mbabker** [comment]
So, a lot of us following this tracker and doing the heavy lifting on the UI stuff aren't as well versed in accessibility matters as some on the accessibility team.  To help us all be able to address these issues, a more clear explanation of what the issue is and how it can be addressed would really be helpful.  Linking to the W3C docs helps explain the concepts at a high level, but for someone like me who is predominantly a backend person who does enough frontend stuff to keep things moving, trying to work through those docs to actually understand why what's there now is wrong and how to fix it isn't the easiest thing to do.

**Comment by ghost** [comment]
closed as having Pull Request #21002
